makiflow/base/maki_entities/input_maki_layer.py

```diff
@@ -60,5 +60,15 @@ def get_params_dict(self):
         """
         return self._named_params_dict
 
+    def get_params_regularize(self):
+        """
+        This code imitates MakiLayer API for the compatibility sake.
+        This data is used for collect params for regularisation.
```

Comment:
Add explanation for why this method actually exists. 
Some of the parameters, like bias, are preferred not to be regularized since it can cause underfitting. Thus, it makes sense to track parameters that are being regularized and that are not.